client/web/src/components/widgets/floating-menus/MenuList.vue

```diff
@@ -15,7 +15,16 @@
 				<Icon :icon="entry.icon" v-if="entry.icon && drawIcon" />
 				<div class="no-icon" v-else-if="drawIcon" />
 				<span class="entry-label">{{ entry.label }}</span>
-				<UserInputLabel v-if="entry.shortcut && entry.shortcut.length" :inputKeys="[entry.shortcut]" />
+				<Icon
+					:icon="'Info'"
+					v-if="entry.shortcutRequiresLock && !fullscreen.keyboardLocked"
+					:title="
+						keyboardLockApiSupported
+							? 'This hotkey is reserved by the browser, but becomes available in fullscreen mode'
+							: 'This hotkey is reserved by the browser, but becomes available in Chrome, Edge, and Opera which support the Keyboard.lock() API'
+					"
+				/>
+				<UserInputLabel v-else-if="entry.shortcut && entry.shortcut.length" :inputKeys="[entry.shortcut]" />
```

Comment:
If typescript is allowed in these attributes, you may be able to do something cleaner like `entry.shortcut?.length`

Reply:
Infuriatingly, WebPack doesn't let us use the optional chaining operator.